models/src/main/thrift/content/v1.thrift

```diff
@@ -1119,9 +1119,7 @@ struct Section {
 struct Atoms {
     1: optional list<contentatom.Atom> quizzes
 
-    2: optional list<contentatom.Atom> viewpoints
-
-    3: optional list<contentatom.Atom> media
+    2: optional list<contentatom.Atom> media
```

Comment:
Let's keep `media` as field number 3 and leave the viewpoints line commented out as a reminder that we should never reuse field number 2. Reusing field numbers is a big no-no in Thrift.